app/models/concerns/callable.rb

```diff
@@ -10,6 +10,10 @@ def old_calls
     calls.includes(:created_by).order('created_at DESC').offset(10)
   end
 
+  def last_call
```

Comment:
This is pretty much just a drop-in replacement for `calls.sort_by(created_at).last` in `days_since_last_call`. Makes it look a little cleaner, I thought.